README.md

```diff
@@ -1,2 +1,335 @@
-docket is a distributed background task system for Python functions with a focus
+Docket is a distributed background task system for Python functions with a focus
 on the scheduling of future work as seamlessly and efficiency as immediate work.
+
+[![PyPI - Version](https://img.shields.io/pypi/v/pydocket)](https://pypi.org/project/pydocket/)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/pydocket)](https://pypi.org/project/pydocket/)
+[![GitHub main checks](https://img.shields.io/github/check-runs/chrisguidry/docket/main)](https://github.com/chrisguidry/docket/actions/workflows/ci.yml)
+[![Codecov](https://img.shields.io/codecov/c/github/chrisguidry/docket)](https://app.codecov.io/gh/chrisguidry/docket)
+[![PyPI - License](https://img.shields.io/pypi/l/pydocket)](https://github.com/chrisguidry/docket/blob/main/LICENSE)
+
+## At a glance
+
+```python
+from datetime import datetime, timedelta, timezone
+
+from docket import Docket, Worker
+
+
+async def greet(name: str, greeting="Hello") -> None:
+    print(f"{greeting}, {name} at {datetime.now()}!")
+
+
+async with Docket() as docket:
+    await docket.add(greet)("Jane")
+
+    now = datetime.now(timezone.utc)
+    soon = now + timedelta(seconds=3)
+    await docket.add(greet, when=soon)("John", greeting="Howdy")
+
+    async with Worker(docket) as worker:
+        await worker.run_until_finished()
+```
+
+```
+Hello, Jane at 2025-03-05 13:58:21.552644!
+Howdy, John at 2025-03-05 13:58:24.550773!
+```
```

Comment:
Not sure if you want to split it up this way, but may make the distributed nature more clear?

```python
from datetime import datetime, timedelta, timezone

from docket import Docket

async def greet(name: str, greeting="Hello") -> None:
    print(f"{greeting}, {name} at {datetime.now()}!")


async with Docket() as docket:
    await docket.add(greet)("Jane")

    now = datetime.now(timezone.utc)
    soon = now + timedelta(seconds=3)
    await docket.add(greet, when=soon)("John", greeting="Howdy")
```
```python
from docket import Docket, Worker
    async with Worker(docket) as worker:
        await worker.run_until_finished()
```

```
Hello, Jane at 2025-03-05 13:58:21.552644!
Howdy, John at 2025-03-05 13:58:24.550773!
```